tracer/src/Datadog.Trace.Trimming/build/Datadog.Trace.Trimming.xml

```diff
@@ -990,6 +990,9 @@
       <type fullname="System.Threading.Timer" />
       <type fullname="System.Threading.TimerCallback" />
    </assembly>
+   <assembly fullname="System.Web.HttpUtility">
```

Comment:
I'm not clear on why this change is here, can we revert it?

Reply:
There's a new call to `HttpUtility.ParseQueryString()` in `AgentBatchUploadApi.cs`. This file is updated automatically so the `HttpUtility` class is not trimmed away if users use the SDK's trimming feature.

Reply:
Got it, makes sense, thanks for explaining!